src/java.base/share/classes/java/nio/channels/DatagramChannel.java

```diff
@@ -172,8 +172,7 @@ public static DatagramChannel open() throws IOException {
      * java.nio.channels.spi.SelectorProvider} object.  The channel will not be
      * connected.
      *
-     * @apiNote {@linkplain java.net.UnixDomainSocketAddress Unix domain} sockets
-     * are not supported by DatagramChannel.
+     * @apiNote {@link java.net.StandardProtocolFamily#UNIX} is not supported.
```

Comment:
It might be clearer to restore the original text ("Unix domain sockets are not support by DatagramChannel") and add something like "; invoking this method with the UNIX protocol family throws UnsupportedOperationException". 

(btw: I had changed the description of the JBS issue to reflect what this issue about, hopefully the commit can use the same description).

Reply:
I've updated the PR to be the same as the bug report. But, is it necessary to mention UOE, given that the @throws already says:
```
     * @throws  UnsupportedOperationException
     *          If the specified protocol family is not supported. For example,
     *          suppose the parameter is specified as {@link
     *          java.net.StandardProtocolFamily#INET6 StandardProtocolFamily.INET6}
     *          but IPv6 is not enabled on the platform.
```

Reply:
How about the following?
```
     * @apiNote {@linkplain java.net.StandardProtocolFamily#UNIX Unix domain}
     *          sockets are not supported.
```

Reply:
My concern with linking to UDSA is that it's a SocketAddress and the parameter is a ProtocolFamily. So if there is a link then I think it should be to StandardProtocolFamily#UNIX. The text above 'Unix domain sockets are not supported by DatagramSocket" without a link is okay too.